shell/platform/darwin/macos/framework/Source/FLEEngine.mm

```diff
@@ -345,6 +345,49 @@ - (void)sendOnChannel:(nonnull NSString*)channel message:(nullable NSData*)messa
   }
```

Comment:
I would expect sendOnChannel:message: to be implemented by calling sendOnChannel:message:binaryReply: with a nil reply callback, so that we don't have duplicate codepaths.

Reply:
Done